ezflow/data/dataset/base_dataset.py

```diff
@@ -133,6 +137,15 @@ def __getitem__(self, index):
         img2 = torch.from_numpy(img2).permute(2, 0, 1).float()
         flow = torch.from_numpy(flow).permute(2, 0, 1).float()
 
+        if self.normalize:
+            #input_transform = transforms.Compose([
+            #transforms.Normalize(mean=[0,0,0], std=[255,255,255]),
+            #])
+            #img1 = input_transform(img1)
+            #img2 = input_transform(img2)
+            img1 = normalize_image(img1)
```

Comment:
let's make this operation in a single step.

For example: `img1, img2 = normalize_image(img1, img2)`

Reply:
Done.